src/jstests/formio/validators/mocks.js

```diff
@@ -0,0 +1,31 @@
+import {rest} from 'msw';
+
+import {BASE_URL} from 'api-mocks';
+
+const INTERNATIONAL_VALIDATION_ENDPOINT = `${BASE_URL}validation/plugins/phonenumber-international`;
+const DUTCH_VALIDATION_ENDPOINT = `${BASE_URL}validation/plugins/phonenumber-nl`;
+
+export const mockInternationalValidPost = rest.post(
```

Comment:
I'd prefer to have the validator names in here - `mockValidInternationalPhonenumberPost` - if we add other validators in the future, then the names won't collide.

You could also namespace this:

```js
export const phoneNumberValidations = {
  internationalValidPost: ...,
  internationalInvalidPost: ...,
  dutchValidPost: ...,
  dutchInvalidPost: ...,
};
```